kani-compiler/src/codegen_cprover_gotoc/codegen/block.rs

```diff
@@ -3,8 +3,13 @@
 
 use crate::codegen_cprover_gotoc::GotocCtx;
 use rustc_middle::mir::{BasicBlock, BasicBlockData};
+use stable_mir::mir::BasicBlockIdx;
 use tracing::debug;
 
+pub fn bb_label(bb: BasicBlockIdx) -> String {
+    format!("bb{bb}")
+}
+
```

Comment:
We still have a few usages of `find_label()` which provides the internal version of `BasicBlockIdx`. Following the pattern that I've been trying to follow would be to create another function named `find_label_stable()` which takes the new type. However, I decided to create a function that is much simpler and that has a more meaningful name. The original `find_label` doesn't find any label and it doesn't need access to the function context. So the new usages are much  simpler, and instead of rewriting them to be something like:

```rust
self.function_ctx().find_label_stable(bb)
```
we just use:
```rust
bb_label(bb)
```

Reply:
Ok, so you need both for now but won't need `find_label` once you've completed the migration. Sounds good!